tests/cephadm/test_remove_service.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class RemoveService(Exception):
+    pass
+
+
+def _verify_service(node, service_name, present=True):
+    """
+    Verifies service is removed
+    Args:
+        node (str): monitor node object
+        service_name (str): service name
+    return: (bool)
+    """
+    # Get service status
+    out = CephAdm(node).ceph.orch.ls(service_type=service_name, format="json-pretty")
+
+    # If service was deployed
+    if present:
+        service_details = json.loads(out)[0]
+        return (
+            False
+            if "service was created" not in (service_details["events"][0])
+            else True
+        )
+    else:
+        # If service was removed, wait for a timeout to check whether its removed
+        timeout, interval = 20, 2
+        for w in WaitUntil(timeout=timeout, interval=interval):
+            out = CephAdm(node).ceph.orch.ls(
+                service_type=service_name, format="json-pretty"
+            )
+            if "No services reported" in out:
+                return True
+        if w.expired:
+            log.info(f"Service {service_name} is not removed after rm operation.")
+    return False
+
+
+def run(ceph_cluster, **kw):
+    """Verify redeploy for a specific service
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+    """
+    config = kw.get("config")
+    node = ceph_cluster.get_nodes(role="mon")[0]
+    service_name = config.get("service_name")
```

Comment:
Same as above, add validation for configs

Reply:
Updated

Reply:
I don't see any validation for config here

Reply:
Updated